Playground/SwiftShortcutsTemplate.playgroundbook/Contents/Manifest.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Chapters</key>
+	<array>
+		<string>Chapter1.playgroundchapter</string>
+	</array>
+	<key>ContentIdentifier</key>
+	<string>ios.github.a2.swift-shortcuts</string>
```

Comment:
For consistency with [the old version](https://github.com/a2/shortcuts-swift/blob/45f825c584dccafb0a1cf2fa836404fbaf49ecaf/feed.json#L13) of the library, can we use the following?

```suggestion
	<string>io.a2.swift-shortcuts</string>
```

Reply:
Ahh I didn't realise that you had a playground before, nice!